### PR TITLE
Feature/survey route

### PR DESCRIPTION
**Pull request by DrewBradley**
### Fix/Feature/Bug?
Feature

### What does this PR do?
Adds a route for the survey component

### Where should review start?
Lines 13-17 of code/web/src/setup/routes/crate.js

### How is this tested?
NA

### Applicable tickets?
NA

### CODE REVIEW

File: code/web/src/setup/routes/user.js [hunk 1]
```diff
@@ -26,5 +27,11 @@ export default {
     path: '/user/subscriptions',
     component: Subscriptions,
     auth: true
+  },
+
+  survey: {
+    path: '/user/survey',
+    component: Survey,
+    auth: true
```
Comment (deadbelly) [inline code/web/src/setup/routes/user.js]: So this is a simple little block of code and it looks fine, but I just want to confirm: if we're using a new path for the survey is it still appearing in a modal? Or are we now re-routing the user to a new page?

**Review by deadbelly — CHANGES_REQUESTED** [review]
Looks great but I've got some questions about whether we're deviating from our plan a bit. I'm ok with it either way.

File: code/web/src/setup/routes/user.js (hunk 1 above)
Comment (DrewBradley) [reply code/web/src/setup/routes/user.js]: I think it does reroute to a new page. In which case, we should NOT create this extra route.

I was trying to add this so we could view the survey while we were building it, but I don't think we need to.

We can reject and close this PR.

Comment (deadbelly) [reply code/web/src/setup/routes/user.js]: We can always add the route on top of `/crates` right? So `/crates` might let you pick from several crates to subscribe to, and then when you add a subscription you're routed to `/crates/survey`? Or maybe `/crates/${crate-id}-survey`? I'll close this PR if you want but just wanted to throw that out there.

Comment (DrewBradley) [reply code/web/src/setup/routes/user.js]: That seems like a good idea. I will make those changes.

File: code/web/src/setup/routes/crate.js [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
 // Crate routes
 export default {
   list: {
     path: '/crates',
     component: List,
     auth: true
+  },
+
+  survey: {
+    path: '/crates/${crate-id}-survey',
```
Comment (deadbelly) [inline code/web/src/setup/routes/crate.js]: I think you'd want to use backticks ``` to interpolate `${crate-id}` but I also think that for now we should just use `/crate/survey` and we can make it more specific if it helps error handling later.

**Review by deadbelly — CHANGES_REQUESTED** [review]

File: code/web/src/setup/routes/crate.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ export default {
   },
 
   survey: {
-    path: '/crates/${crate-id}-survey',
+    path: '/crates/survey',
```
Comment (deadbelly) [inline code/web/src/setup/routes/crate.js:14]: This looks great Ill approve this PR! Thanks for going through those changes!

**Review by deadbelly — APPROVED** [review]

**Pull request merged**